lib/Workflow/Condition/CheckReturn.pm

```diff
@@ -63,11 +62,14 @@ sub evaluate {
     } elsif ( $arg =~ /^[a-zA-Z0-9_]+$/ ) {    # alpha-numeric, plus '_'
         $argval = $wf->context->param($arg);
     } else {
+        local $@;
```

Comment:
What is the scope for `local` ?

Reply:
The scope is from the `local` statement to the end of the lexical block. At the end of the lexical scope, the old variables get restored. (Note that a `local $var` assigns "undef" to var.)